Robot Code/robot.py

```diff
@@ -1,77 +1,56 @@
 #!/usr/bin/env python3
-# Python3 Robot code: "Unititled"
-# 2018 - 2018, 4480 "Forty48ie" "UC-Botics"
 #
+# Python3 Robot code: "Unititled" using Robotpy
+# 2018 - 2018, 4480 "Forty48ie" "UC-Botics"
+# This Code is protected by UC-Botics under the leadership of our profound and prominent
+# Senior Mechanical Engineer: Ethan Robertson and hereby stands as an inspirations to all
+# of us here to do great things.
 
 import wpilib
 import wpilib.buttons
-from robotpy_ext.autonomous import AutonomousModeSelector
-from robotpy_ext.common_drivers import units, navx
-import networktables
-from wpilib.drive import DifferentialDrive
-import wpilib.drive
-import math
-import hal
-
-
-print (wpilib.__version__)
-
+import ctre
+from wpilib.drive import DifferentialDrive        
 class MyRobot(wpilib.IterativeRobot):
 
 
     def robotInit(self):
 
-
-        #Motors
-        self.leftMotorInput = wpilib.Talon(1) #  AEN
-        self.rightMotorInput = wpilib.Talon(2) # AEN
-     
-        self.drive = wpilib.drive.DifferentialDrive(self.leftMotorInput, self.rightMotorInput)
-      
-      
-        #Inputs
+        #Drive Motors
+        self.motor1 = ctre.WPI_TalonSRX(1)
+        self.motor2 = ctre.WPI_TalonSRX(2)
```

Comment:
Use follower mode for a speed controller each side http://robotpy.readthedocs.io/projects/ctre/en/latest/api_talonsrx.html#ctre.talonsrx.TalonSRX.follow